command/app.go

```diff
@@ -55,9 +55,14 @@ var app = &cli.App{
 			Name:  "install-completion",
 			Usage: "install completion for your shell",
 		},
+		&cli.BoolFlag{
+			Name:  "dry-run",
```

Comment:
why --dry-run is an app flag? not all operations have a side effect. we discussed it before but I could not find a reference to our previous discussion.

Reply:
we had discussed it with @ilkinulas that making dry-run global might be better when moving its implementation to storage. For example, someone might want to perform `s5cmd --dry-run run commands.txt` and it encompasses all subcommands that might have a side-effect